src/core/public/chrome/chrome_service.tsx

```diff
@@ -164,6 +164,15 @@ export class ChromeService {
       );
     }
 
+    const settings = injectedMetadata.getLegacyMetadata().uiSettings;
+
+    const navSetting =
+      settings?.user?.pageNavigation?.userValue || settings.defaults.pageNavigation.value;
+
+    const license = (injectedMetadata.getInjectedVars()?.xpackInitialInfo as {
+      license: { type: string };
+    })?.license?.type;
```

Comment:
I thought the license check code was removed?

Reply:
Sorry, hadn't pushed these changes before your review